src/common/services/request/query/fetch-plays-filter.js

```diff
@@ -0,0 +1,265 @@
+export const FetchPlaysFilter = {
+  // Filter all the featured plays
+  getAllFeaturedPlays: () => {
+    return {
+      display: "Filter all the featured plays",
+      name: "Fetch_Plays",
+      function: "plays",
+      write: false,
+      params: [
+        "blog",
+        "component",
+        "cover",
+        "created_at",
```

Comment:
Actually, we use all these parameters as react-snap require them at the route level to create SSR. Without passing them, the SSR for each URL will not be created for each of the play URL and thus it will not create the social previews of the play. SO we need these fields to be uniformly retrieved for all plays(or paginated), and filtered plays.

One improvement that I discussed with @Angryman18 was the place where the fetch query happen today is not right. We are making the fetch when we are on the `/` route but we should make it on the `/plays` route. This fix has to be taken care but we can do that immediately after this PR.

Reply:
It would be good if we introduce dynamic data loading/routing .

Reply:
It would be good if we introduce dynamic data loading/routing .